src/t2iapi/biceps/localizedtext.proto

```diff
@@ -0,0 +1,31 @@
+/*
+This Source Code Form is subject to the terms of the MIT License.
+Copyright (c) 2024 Draegerwerk AG & Co. KGaA.
+
+SPDX-License-Identifier: MIT
+*/
+
+syntax = "proto3";
+
+package t2iapi.biceps;
+
+option java_package = "com.draeger.medical.t2iapi.biceps";
+option java_outer_classname = "LocalizedTextProto";
+
+import "t2iapi/biceps/localizedtextcontent.proto";
+import "t2iapi/biceps/localizedtextref.proto";
+import "google/protobuf/wrappers.proto";
+import "t2iapi/biceps/referencedversion.proto";
+import "t2iapi/biceps/localizedtextwidth.proto";
+
+/*
+Represents pm:LocalizedText (defined in IEEE Std 11073-10207-2017).
+*/
+message LocalizedTextMsg {
+  LocalizedTextContentMsg localized_text_content = 1;
+  LocalizedTextRefMsg ref_attr = 2;
+  google.protobuf.StringValue lang_attr = 3;
+  ReferencedVersionMsg version_attr = 4;
```

Comment:
No need for this type indirection

Reply:
done - removed the indirection, and set it directly to uint64

Reply:
Should be [UInt64Value](https://protobuf.dev/reference/protobuf/google.protobuf/#uint64-value) though.

Reply:
done